Sources/StepperView/Extension/EnvironmentValues+Extension.swift

```diff
@@ -167,3 +174,11 @@ struct StepLifeCycleKey: EnvironmentKey {
     /// provide a default value for custom dependency
     static var defaultValue:[StepLifeCycle] = []
 }
+
+/// Environment Key for StepperEdgeInsets .
+@available(iOS 13.0, OSX 10.15, tvOS 13.0, watchOS 6.0, *)
+struct StepperEdgeInsetsKey: EnvironmentKey {
+    /// provide default values for Stepper Edge Insets
+    static var defaultValue: EdgeInsets =
+        EdgeInsets(top: Utils.standardSpacing, leading: Utils.standardSpacing, bottom: Utils.standardSpacing, trailing: Utils.standardSpacing)
```

Comment:
This may slightly change the default spacing, as if you just add `.padding()` it uses the system default (varies based on device type and other conditions). If we made `stepperEdgeInsets` be optional `EdgeInsets?` with a default value of `nil` then passing `.padding(nil)` will retain the current behavior.

Reply:
I am evaluating this behaviour across all the watchOS, iOS